src/code/reducers/index.js

```diff
@@ -31,6 +31,63 @@ function initialState() {
           });
 }
 
+function fertilize(state) {
```

Comment:
The `fertilize` function uses `state.gametes` to create an offspring given user-created gametes.

If you look at the state during an egg game, you'll see something like

```
gametes: {
    currentGametes: [
      {
        '1': 'a',
        '2': 'b',
        XY: 'y'
      },
      {
        '1': 'b',
        '2': 'b',
        XY: 'x2'
      }
    ],
```

these are the sides that the user (or possibly the randomization function) has selected for each chromosome pair, for each gamete.

These together represent the entirety of the offspring. There is no more randomization. The annoying psrt (all this code) is to go from that list of sides to an organism. We have to get all the chromosomes, get the appropriate side, and concatenate all the alleles together. I don't believe that there is a Biologica function to do this, but it could certainly be added.

But unless I'm missing something, I would *not* use this for the clutch breeding. Instead I would just use `BioLogica.breed(org1, org2)`.

You can see an example of this in the non-Redux example `/examples/basic-examples/breeding.html` (http://concord-consortium.github.io/geniblocks/examples/basic-examples/breeding.html). I believe I added it as one of the first Redux actions way at the start, but then we removed it because nothing (yet) was using it.